pkgs/development/python-modules/jupytext/default.nix

```
@@ -1,39 +1,47 @@
-{ lib, buildPythonPackage, fetchPypi, isPy27
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, pytestCheckHook
+, GitPython
+, jupyter_client
 , markdown-it-py
+, mdit-py-plugins
 , nbformat
-, pytest
+, notebook
 , pyyaml
 , toml
 }:
 
 buildPythonPackage rec {
   pname = "jupytext";
-  version = "1.11.0";
+  version = "1.11.2";
 
   disabled = isPy27;
```

See https://github.com/mwouts/jupytext/blob/master/setup.py#L41

```suggestion
  disabled = pythonOlder "3.6";
```